chrome/renderer/chrome_content_renderer_client.cc

```diff
@@ -381,6 +385,15 @@ void ChromeContentRendererClient::RenderThreadStarted() {
           ? blink::scheduler::RendererProcessType::kExtensionRenderer
           : blink::scheduler::RendererProcessType::kRenderer);
 
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
```

Comment:
It can cause unused variable error. Please move into CASTANETS block.

Reply:
This one was moved from line 446, and 459, 461 is using it. So I think that it is not goint to be warned as unused variable build warning.

Reply:
Oh, I see.